pcap/dlt.h

```diff
@@ -1426,6 +1426,121 @@
  */
 #define DLT_VPP_DISPATCH	280
 
+/*
+ * Broadcom Ethernet switches (ROBO switch) 4 bytes proprietary tagging
+ * format.
+ *
```

Comment:
I've checked in a change to the link-layer header types page on tcpdump.org to add the two Broadcom switch tag LINKTYPE_/DLT_ values, along with a page that describes the Broadcom switch tag formats, so you don't need to have the switch tag format description here any more.  Those changes should show up on the Web site within 24 hours or so.